server/index.js

```diff
@@ -1,131 +1,21 @@
 const fileUpload = require('express-fileupload')
 const express = require('express')
 const path = require('path')
-const fs = require('fs')
-const rimraf = require('rimraf')
 
 const app = express()
 
-const { isFileAttached } = require('./utils/middlewares')
-const { getAllFiles } = require('./utils/files.js')
-
 // default options
 app.use(fileUpload())
 
 app.use(express.static(path.join(__dirname, '..', '/client/build')))
 
-const defaultLocation = '..'
-
-// Get list of all files
-app.get('/files', (req, res) => {
-	return getAllFiles(`${defaultLocation}/snippets`).then(files =>
-		res.send({ files })
-	)
-})
+// Routers
+const labelRouter = require('./routes/label');
+const snippetRouter = require('./routes/snippet');
```

Comment:
Can you rename the imports to just `label` and `snippet`?

Reply:
Yes, will do.